src/client/Microsoft.Identity.Client/AppConfig/PopAuthenticationConfiguration.cs

```diff
@@ -21,8 +21,8 @@ namespace Microsoft.Identity.Client.AppConfig
     public class PoPAuthenticationConfiguration
     {
         /// <summary>
-        /// Creates a configuration using the default key management. Use the properties <see cref="HttpMethod"/>, <see cref="HttpHost"/> etc. 
-        /// to control which elements of the request should be included in the POP token.
+        /// Creates a configuration using the default key management - an RSA key will be created in memory and rotated every 8h.
```

Comment:
Unrelated - why 8h?

Reply:
SAL uses 12h as default. We picked 8h more or less for it to be similar to SAL. 
After 8h, a new key is generated and the pop envelope from the cache becomes invalid, i.e. a new request to AAD is needed.

Reply:
The default should take into account: 
- token lifetime (24h nowadays I think for long lived tokens)
- token refresh (1/2 of lifetime, so 12h)

I think anything lower the token refresh interval is pretty good, to avoid having to refresh because of refrsh_in and then refresh because of POP rotation interval.

Great question.